retrofit/lib/client/client.dart

```diff
@@ -6,45 +6,32 @@ import 'package:jaguar_resty/jaguar_resty.dart';
 
 abstract class ApiClient {
   Route get base;
-  JsonRepo get jsonConverter;
+  static const contentTypeJson = 'application/json';
```

Comment:
It is not good to declare them here. I think the best way to get these constants is `dart:io`'s ContentType class. If that is not good enough, we can create a `ContentType` class and put it in resty.

Reply:
ho didn't know it was available on `dart:io` ! I'll change to use those instead then ! much better thanks !

Reply:
:( unfortunately we can't reuse the `dart:io ` one :( annotation can only call `const` constructor and `ContentType` is not const :( so I'll create a class in resty for that

Reply:
Done and pushed @tejainece :) much better like this indeed !